apps/workflowengine/templates/settings.php

```diff
@@ -22,22 +22,4 @@
 /** @var array $_ */
 /** @var \OCP\IL10N $l */
 ?>
-<div id="<?php p($_['appid']); ?>" class="section workflowengine">
-	<h2 class="inlineblock"><?php p($_['heading']); ?></h2>
-	<?php if (!empty($_['docs'])): ?>
-		<a target="_blank" rel="noreferrer noopener" class="icon-info svg"
-		   title="<?php p($l->t('Open documentation'));?>"
-		   href="<?php p(link_to_docs($_['docs'])); ?>">
-		</a>
-	<?php endif; ?>
-
-	<?php if (!empty($_['settings-hint'])): ?>
-		<p class="settings-hint"><?php p($_['settings-hint']); ?></p>
-	<?php endif; ?>
-
-	<?php if (!empty($_['description'])): ?>
-		<p><?php p($_['description']); ?></p>
-	<?php endif; ?>
-
-	<div class="rules"><span class="icon-loading-small"></span> <?php p($l->t('Loading…')); ?></div>
-</div>
+<div id="<? p(\OCA\WorkflowEngine\AppInfo\Application::APP_ID); ?>"></div>
```

Comment:
```suggestion
<div id="<?php p(\OCA\WorkflowEngine\AppInfo\Application::APP_ID); ?>"></div>
```

it does not render short tags for me, also they are deprecated https://wiki.php.net/rfc/deprecate_php_short_tags